.github/workflows/weekly.yml

```diff
@@ -22,36 +22,24 @@ on:
     - cron: "0 0 * * 0"
   workflow_dispatch:
 jobs:
-  # tests that current connector iteration does not break compatibility with last 2 minor released Flink versions
-  non-main-version:
+  compile_and_test:
     if: github.repository_owner == 'apache'
+    strategy:
+      matrix:
+        flink_branches: [{
```

Comment:
It is true that all the PRs are tested against last 2 Flink versions but don't you think that the weekly workflow should also test against these versions and not only the snapshots ?

Reply:
perfect :+1: